packages/client/modules/teamDashboard/components/AgendaList/AgendaList.tsx

```diff
@@ -86,7 +97,7 @@ const AgendaList = (props: Props) => {
       <Droppable droppableId={AGENDA_ITEM}>
         {(provided) => {
           return (
-            <AgendaListRoot ref={provided.innerRef}>
+            <AgendaListRoot onMouseLeave={handleMouseLeave} ref={provided.innerRef}>
```

Comment:
from a high level, I'd think the hover state of an item should be encapsulated within the item.
is it necessary to put the hover state on the list?

Reply:
Agreed - I reluctantly went for this approach because `onMouseLeave` wasn't being triggered when the cursor went through the tooltip and out of the item. With the `pointerEvent` fix, I've been able to remove this